huaweicloud/resource_huaweicloud_networking_secgroup_test.go

```diff
@@ -8,13 +8,13 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
 
-	"github.com/chnsz/golangsdk/openstack/networking/v3/security/groups"
+	"github.com/chnsz/golangsdk/openstack/networking/v1/security/securitygroups"
```

Comment:
maybe you can just add an alias `group` next time

Reply:
ok